src/core/api/search.go

```diff
@@ -95,8 +96,12 @@ func (s *SearchAPI) Get() {
 			continue
 		}
 
-		if isAuthenticated {
-			roles := s.SecurityCtx.GetProjectRoles(p.ProjectID)
+		if sc, ok := s.SecurityCtx.(*local.SecurityContext); ok {
```

Comment:
Is this intentional? Previously it doesn't require the securityContext must be local based.